idd/Energy+.idd.in

```diff
@@ -40777,6 +40786,7 @@ ZoneHVAC:EquipmentList,
        \key AirTerminal:SingleDuct:Uncontrolled
        \key ZoneHVAC:EnergyRecoveryVentilator
        \key ZoneHVAC:EvaporativeCoolerUnit
+       \key ZoneHVAC:ForcedAir:UserDefined
```

Comment:
Ack, this is OK, but the Zone Equipment n Name fields don't have an object-list reference.  Would be nice to add that to all of these equipment types.